ironfish/src/rpc/clients/memoryClient.test.ts

```diff
@@ -18,10 +18,12 @@ describe('MemoryClient', () => {
         networkId: 2,
       },
     })
-    const client = new RpcMemoryClient(createRootLogger(), await sdk.node())
+    const client = new RpcMemoryClient(
+      createRootLogger(),
+      (await sdk.node()).rpc.getRouter(ALL_API_NAMESPACES),
+    )
 
-    const allowedNamespaces = ALL_API_NAMESPACES
-    const loadedNamespaces = [...(client.router?.routes.routes.keys() || [])]
-    expect([...allowedNamespaces.values()].sort()).toMatchObject(loadedNamespaces.sort())
+    const loadedNamespaces = [...(client.router.routes.routes.keys() || [])]
+    expect([...ALL_API_NAMESPACES.values()].sort()).toMatchObject(loadedNamespaces.sort())
```

Comment:
This test should be deleted now, because it's now a test for the router itself which doesn't belong in something that just takes a router.

Reply:
Yea I'll remove this